concent-builder/configure.yml

```diff
@@ -14,6 +14,11 @@
             - ca-certificates
             - curl
 
+        - name:    Add debian test repository for python3.6 installation
+          apt_repository:
+            repo:  "deb http://ftp.de.debian.org/debian testing main"
+            state: present
```

Comment:
This permanently adds the testing repo. We don't want that. We only want to install some stuff from testing. Use the same method I suggested for the Dockerfile.